### ISSUE TITLE
Could not get Accessibility View to work with extension contributed hovers

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#186296]
Testing #186214

1. Install GitHub Pull Request extension. Open GitHub view on the side. Hover over an issue, and put focus in it
2. Ctrl + F2 to open Accessibility View. Nothing happens 🐛 

I could not get the Accessibility View to work with any hover. What am I missing here?

**Comment by meganrogge** [comment]
Hm, any errors in the dev tools console?
https://github.com/microsoft/vscode/assets/29464607/4ee829a1-afd3-4217-b048-f9e51039bef0

**Comment by meganrogge** [comment]
Yeah they're not working in the scm view for me

**Comment by isidorn** [comment]
This still does not work for me for the GitHub issues view.
@meganrogge can you reproduce with the GitHub issues view? You should just install the GH PR extension and you should get the issues view and you can try out this there. Thanks

**Comment by meganrogge** [comment]
Here's what I see in the GH PR view - I'll try issues 
https://github.com/microsoft/vscode/assets/29464607/037666b3-13ca-4656-9ed3-97c8a9b725e4

**Comment by meganrogge** [comment]
it's also working for me in the github issues view - I'm using the keyboard shortcut of `cmd + k + i`. are you? 
Uploading issues.mov…

**Comment by isidorn** [comment]
Works now. I was doing something wrong. Thank you. Adding verified.
